site/en/guide/distributed_training.ipynb

```diff
@@ -636,7 +636,7 @@
       "source": [
         "## Use tf.distribute.Strategy with custom training loops\n",
         "\n",
-        "As demonstrated above, using `tf.distribute.Strategy` with Keras `Model.fit` requires changing only a couple lines of your code. With a little more effort, you can also use `tf.distribute.Strategy` [with custom training loops](/keras/writing_a_training_loop_from_scratch.ipynb).\n",
+        "As demonstrated above, using `tf.distribute.Strategy` with Keras `Model.fit` requires changing only a couple lines of your code. With a little more effort, you can also use `tf.distribute.Strategy` [with custom training loops](keras-io/guides/ipynb/writing_a_training_loop_from_scratch.ipynb).\n",
```

Comment:
Relative linking (https://www.tensorflow.org/community/contribute/docs_style#links_in_markdown_and_notebooks)

Let's try the following:

```suggestion
        "As demonstrated above, using `tf.distribute.Strategy` with Keras `Model.fit` requires changing only a couple lines of your code. With a little more effort, you can also use `tf.distribute.Strategy` [with custom training loops](./keras/writing_a_training_loop_from_scratch.ipynb).\n",
```